conductr_cli/resolvers/bintray_resolver.py

```diff
@@ -142,7 +142,7 @@ def continuous_delivery_uri(resolved_version):
 
 def is_local_file(uri):
     parsed = urlparse(uri, scheme='file')
-    return parsed.scheme == 'file' and parsed.path.endswith('.zip') and os.path.exists(uri)
+    return parsed.scheme == 'file' and os.path.exists(parsed.path)
```

Comment:
We can load many kinds of local files now so we don't want to be as restrictive.